ui/src/components/schemas/SchemaDetails.tsx

```diff
@@ -0,0 +1,340 @@
+                              void message.success("Schema successfully downloaded");
+                            })
+                            .catch(() => {
+                              void message.error(
+                                "An error occurred while downloading the schema. Please try again"
+                              );
+                            });
+                        }}
+                        style={{ height: 24, padding: 0 }}
+                        type="link"
+                      >
+                        JSON Schema
+                      </Button>
+                    </Row>
+                  </Space>
+                </Col>
+
+                {(() => {
+                  switch (jsonView) {
+                    case "formatted": {
+                      return (
+                        <Col style={{ background: "#F2F4F7", borderRadius: 8, padding: 24 }}>
+                          <Space direction="vertical">
+                            <Row justify="space-between">
+                              <Typography.Text type="secondary">ATTRIBUTES</Typography.Text>
+                            </Row>
+                            <SchemaTree schema={schema.data} style={{ background: "#F2F4F7" }} />
+                          </Space>
+                        </Col>
+                      );
+                    }
+                    case "jsonLdContext": {
+                      return (
+                        <SyntaxHighlighter
+                          customStyle={{
+                            background: "#F2F4F7",
+                            borderRadius: 8,
+                            margin: 0,
+                            padding: 16,
+                          }}
+                          language="json"
+                          style={a11yLight}
+                        >
+                          {JSON.stringify(rawJsonLdContext, null, 2)}
+                        </SyntaxHighlighter>
+                      );
+                    }
+                    case "jsonSchema": {
+                      return (
+                        <SyntaxHighlighter
+                          customStyle={{
+                            background: "#F2F4F7",
+                            borderRadius: 8,
+                            margin: 0,
+                            padding: 16,
+                          }}
+                          language="json"
+                          style={a11yLight}
+                        >
+                          {JSON.stringify(rawJsonSchema, null, 2)}
+                        </SyntaxHighlighter>
+                      );
+                    }
+                  }
+                })()}
+
+                <Row justify="end">
+                  <Space size="middle">
+                    <Button
+                      icon={<CreditCardIcon />}
+                      onClick={() => {
+                        void message.warning("Not yet implemented");
```

Comment:
This should navigate to issueCredentials with the id
```suggestion
                       generatePath(ROUTES.issueCredential.path, {schemaID }
```

Reply:
Right, will change that.

BTW better not to create suggestions that are not committable.

Reply:
Complicate to do a suggestion adding the imports at the top

Reply:
Done!